.changeset/neat-mice-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/ag-grid-theme": patch
+---
+
+Fixed status bar and floating filter styles
+Added `compact` parameter to `useAgGridHelpers` to turn on HD compact density styling when set to true
```

Comment:
Is `useAgGridHelpers` something can be used by consumer?

Reply:
No, but I think we were expecting them to copy/paste it iirc

Reply:
yeah not ideal, will reword